client/views/edit/edit.js

```diff
@@ -32,6 +32,7 @@ Template.edit.events({
     const stale = template.$('select[name=stale_length]').val();
     const isHidden = (template.$('select[name=visibility]').val() === 'hidden');
     const social = (template.$('select[name=social]').val() === 'on');
+    const anonymous = template.$('select[name=anonymous]').val() === 'on'; // Fetch the anonymous value from the dropdown
```

Comment:
@singhsanket143 can you move this comment up?

Reply:
@peter-hank Updated the comments here.